admin_app/src/app/content/page.tsx

```diff
@@ -102,29 +104,73 @@ const CardsPage = () => {
 };
 
 const CardsUtilityStrip = ({ editAccess }: { editAccess: boolean }) => {
+  const [openDownloadModal, setOpenDownloadModal] =
+    React.useState<boolean>(false);
+  const [snackMessage, setSnackMessage] = React.useState<string | null>(null);
+
   return (
-    <Layout.FlexBox
-      key={"utility-strip"}
-      flexDirection={"row"}
-      justifyContent={"flex-right"}
-      alignItems={"right"}
-      sx={{
-        display: "flex",
-        alignSelf: "flex-end",
-        px: sizes.baseGap,
-      }}
-      gap={sizes.baseGap}
-    >
-      <Button
-        variant="contained"
-        disabled={!editAccess}
-        component={Link}
-        href="/content/edit"
-        startIcon={<Add fontSize="small" />}
+    <>
+      <Snackbar
+        open={snackMessage !== null}
+        autoHideDuration={6000}
+        onClose={() => {
+          setSnackMessage(null);
+        }}
       >
-        New
-      </Button>
-    </Layout.FlexBox>
+        <Alert
+          onClose={() => {
+            setSnackMessage(null);
+          }}
+          severity="error"
+          variant="filled"
+          sx={{ width: "100%" }}
+        >
+          {snackMessage}
+        </Alert>
+      </Snackbar>
+      <Layout.FlexBox
+        key={"utility-strip"}
+        flexDirection={"row"}
+        justifyContent={"flex-right"}
+        alignItems={"right"}
+        sx={{
+          display: "flex",
+          alignSelf: "flex-end",
+          px: sizes.baseGap,
+        }}
+        gap={sizes.smallGap}
+      >
+        <Tooltip title="Download all contents">
+          <Button
+            variant="outlined"
+            disabled={!editAccess}
+            onClick={() => {
+              setOpenDownloadModal(true);
+            }}
+          >
+            <DownloadIcon />
+          </Button>
+        </Tooltip>
+        <Tooltip title="Add new content">
+          <Button
+            variant="contained"
+            disabled={!editAccess}
+            component={Link}
+            href="/content/edit"
+            startIcon={<Add />}
+          >
+            New
+          </Button>
+        </Tooltip>
+        <DownloadModal
```

Comment:
Very minor: Can we disable it if there are no cards? 
Ok if you think it's not worth the code

Reply:
Hmm, looked into it and it wasn't trivial - this would involve moving the cards constant up a level etc which might not be worth it. Will add a check to the actual download button to send a snackbar if no data is fetched.